src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const titles = tree.querySelectorAll('li');
+
+titles.forEach(title => {
+  if (title.firstElementChild) {
+    const textNode = title.childNodes[0];
+    const spanTitle = document.createElement('span');
+
+    spanTitle.textContent = textNode.textContent.trim();
+    spanTitle.className = 'tree__title';
+    textNode.textContent = '';
+    title.insertAdjacentElement('afterbegin', spanTitle);
```

Comment:
```suggestion
    textNode.before(spanTitle)
    spanTitle.prepend(textNode)
    spanTitle.className = 'tree__title';
```
Можна додати елемент span і в нього засунути вже існуючий текст чи елемент, замість того щоб видаляти старий